sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.0 (2024-11-05)
 
 ### Other Changes
 
+General Availability release for Azure Data Tables.

```

Comment:
The release notes published along with the blog will end up being empty since they don't include `Other Changes`.

We typically add a summary of the list of features added from the beta releases, aggregating them in the `Features Added` section, since that is visible.

We can use this opportunity to advertise to readers  what is shipping/becoming GA available. This way the end user sees what are the main capabilities relevant to them based on the release notes.

Consider putting something like this in the `Features Added`:
> This update marks the first stable release of the Azure Data Tables library. To use it, reference `Azure::Data::Tables::TableClient` in your applications after including `azure/data/tables.hpp`.

Reply:
the features are already there , i don't see the point in restating the same thing a million times.

Reply:
I understand.

The reason I am suggesting this is because of how the contents of the changelog as used within the release notes and blogs published here:
https://azure.github.io/azure-sdk/releases/2024-10/cpp.html
> The release notes published along with the blog will end up being empty since they don't include `Other Changes`.